lfucg-exactions/server/notes/viewsets.py

```diff
@@ -23,39 +23,62 @@ def get_queryset(self):
 
         child_content_type_string = self.request.query_params.get('content_type', None)
         child_object_id = self.request.query_params.get('object_id', None)
-        parent_content_type_string = self.request.query_params.get('parent_content_type', None)
-        parent_object_id = self.request.query_params.get('parent_object_id', None)
                 
         if child_content_type_string is not None:
             split_child_content = child_content_type_string.split('_')
 
             if len(split_child_content) == 2:
                 child_content_type_app_label = split_child_content[0]
                 child_content_type_model = split_child_content[1]
-
                 child_content_type = ContentType.objects.get(app_label=child_content_type_app_label, model=child_content_type_model)
-
-                if parent_content_type_string is not None:
-                    split_parent_string = parent_content_type_string.split('_')
-
-                    if len(split_parent_string) == 2:
-                        parent_content_type_app_label = split_parent_string[0]
-                        parent_content_type_model = split_parent_string[1]
-
-                        parent_content_type = ContentType.objects.get(app_label=parent_content_type_app_label, model=parent_content_type_model)
-
-                        if parent_content_type and child_content_type:
-                            query_list = queryset.filter(
-                                Q(content_type=parent_content_type, object_id=parent_object_id) |
-                                Q(content_type=child_content_type, object_id=child_object_id))
-
-                        queryset = query_list
-
+                parent_content_type = None
+                parent_object_id = None
+                grandparent_content_type = None
+                grandparent_object_id = None
+                if child_content_type_model == 'lot':
+                    content_lot = Lot.objects.filter(id=child_object_id)
```

Comment:
you can do Lot.objects.filter(id=child_object_id).first(). It will either return the first object or None.